src/components/image/CdrImg.vue

```diff
@@ -25,121 +23,77 @@ const props = defineProps({
-  */
+   * Aspect ratio of the image
+   * @demoSelectMultiple false
+   * @values Valid CSS aspect-ratio property values
+   */
   ratio: {
     type: String,
     default: undefined,
-    validator: (value: string) => ([
-      'auto',
-      'square',
-      '1-2',
-      '2-3',
-      '3-4',
-      '9-16',
-      '2-1',
-      '3-2',
-      '4-3',
-      '16-9'].indexOf(value) >= 0) || false,
   },
   /**
- * Requires a `ratio`. Area to crop the image overflow to. {left, center, right} {top, center, bottom}
- * @demoSelectMultiple false
- * @values left, x-center, right, top, y-center, bottom
- */
-  crop: {
+   * Object position of the image
+   * @demoSelectMultiple false
+   * @values Valid CSS object-position values
+   */
+  position: {
     type: String,
+    default: undefined,
   },
   /**
- * Requires a `ratio`. Scale the image to be as large as possible to fill the area (background-position: cover;)
- */
-  cover: {
-    type: Boolean,
-  },
-  /**
- * Adds a custom class to the cdr-img__ratio container div
- */
-  containerClass: String,
-  /**
- * Sets a border radius to an element {square, top, right, bottom, left}
- * @demoSelectMultiple false
- * @values circle, rounded
- */
-  radius: {
+   * Object fit of the image
+   * @demoSelectMultiple false
+   * @values Valid CSS object-fit values
+   */
+  fit: {
     type: String,
-    validator: (value: string) => ([
-      'circle',
-      'rounded'].indexOf(value) >= 0) || false,
+    default: undefined,
   },
   /**
-   * Modifies the style variant for this component.
+   * Sets border radius
    * @demoSelectMultiple false
-   * @values responsive
+   * @values Accepts shorthand to use cedar radius tokens (soft, softer, round) or a custom value as a valid CSS border-radius values
    */
-  modifier: {
+  radius: {
     type: String,
-    default: '',
-    validator: (value: string) => propValidator(value, ['', 'responsive']),
-  },
+    default: undefined,
+  }
 });
-const style = useCssModule();
-const baseClass = 'cdr-image';
-const ratioClass = 'cdr-image-ratio';
-const coverWrapperClass = 'cdr-image-ratio__cover';
-const modifierClass = computed(() => props.modifier ? `${baseClass}--${props.modifier}` : '');
-const radiusClass = computed(() => props.radius ? `${baseClass}--${props.radius}` : '');
-const cropObject = computed(() => ({ objectPosition: props.crop }));
-const ratioObject = computed(() => {
-  let ratioPct;
-  if (props.ratio === 'square') {
-    ratioPct = '100%';
-  } else if (props.ratio === 'auto') {
-    ratioPct = '0';
-  } else {
-    if (props.ratio) {
-      const [x, y] = props.ratio.split('-');
-      ratioPct = `${(+y / +x) * 100}%`;
-    }
+
+const radiusTokens = 
+  new Map
+    <string | undefined, typeof CdrRadiusSoft | typeof CdrRadiusSofter | typeof CdrRadiusRound>
+      ([
+        ['soft', CdrRadiusSoft],
+        ['softer', CdrRadiusSofter],
+        ['round', CdrRadiusRound]
+      ]); 
```

Comment:
This map allows for a shorthand mapping of CdrToken values. The following computed property allows the component to accept the token or a custom radius.